build-system/tasks/update-packages.js

```diff
@@ -31,7 +31,7 @@ const {isTravisBuild} = require('../common/travis');
 function writeIfUpdated(patchedName, file) {
   if (!fs.existsSync(patchedName) || fs.readFileSync(patchedName) != file) {
     fs.writeFileSync(patchedName, file);
-    if (!isTravisBuild()) {
+    if (!isCiBuild()) {
```

Comment:
General thought after reviewing this PR: a lot of cases where we do these checks are just to determine whether or not to log something. Having a `log` and `ciLog` (or standardizing log levels) could simplify some of the code throughout

Reply:
Agree. Worth considering as a separate PR.